src/sentry/integrations/jira/integration.py

```diff
@@ -253,6 +253,9 @@ def get_link_issue_config(self, group, **kwargs):
     def get_issue_url(self, key, **kwargs):
         return '%s/browse/%s' % (self.model.metadata['base_url'], key)
 
+    def get_persisted_default_config_fields(self):
+        return ['project']
```

Comment:
Will we be adding default issue type as well?

Reply:
I'll need to ask @saragilford I recall her mentioning that wasn't a priority. But I'll double check tomorrow.